sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -20,16 +21,12 @@
 
 
 def _load_allowed_filenames():
-    # Rather than storing the configuration locally,
-    # we should store the configuration somewhere where things
-    # can be changed on the fly.
-    # For example, a dynamodb table or a section in opensearch
-    current_dir = os.path.dirname(__file__)
-    config_file = os.path.join(current_dir, "config.json")
-
-    with open(config_file) as f:
-        data = json.load(f)
-    return data
+    # get the config file from the S3 bucket
+    config_object = s3.get_object(
+        Bucket=f'sds-config-{os.environ["SDSID"]}', Key="config.json"
```

Comment:
What about passing the entire bucket name as the environment variable?
`os.environ["SDS_CONFIG_BUCKET"]` or something similar, so you don't need to keep `sds-config-` hard-coded in both places (in this lambda and below in the CDK stack).

Reply:
Yeah, that would have been smart to do. Updated to pass in the bucket name as an environment variable rather than the sdsid